### PR TITLE
there 2.0.0

### PR DESCRIPTION
**Pull request by morajabi**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/t/there.rb [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
+  url "https://assets-cdn.there.pm/there/There.zip",
+      verified: "github.com/dena-sohrabi/There"
   name "There"
-  desc "Tool to display the local times of people in a team"
+  desc "Tool to display the local times of friends, teammates, cities or any time zone"
   homepage "https://there.pm/"
 
   livecheck do
-    url :url
+    url "https://github.com/dena-sohrabi/There"
```
Comment (daeho-ro) [inline Casks/t/there.rb:12]: I have checked the github link and couldn't find any tags. Then how this livecheck works?

Comment (morajabi) [reply Casks/t/there.rb:12]: I just published a tag. Thanks

**Comment by krehel** [comment]
It seems you've uploaded the app to GitHub.  This set of changes is a regression by eliminating the `sha256`.

**Comment by krehel** [comment]
To add, as I realize now the scope of the changes.  Why is this being moved out of `there-desktop` and into another repo?

**Comment by morajabi** [comment]
It's completely re-written in Swift by @dena-sohrabi. Do you think I should've created a new cask? As the older version hasn't gotten (and won't) get any updates in the last 5 years.
I can also probably ask her to move it to the organization of @therehq if it's a must.

**Comment by morajabi** [comment]
If we can please merge this sooner, I'd love to include the brew install command on our homepage as we're launching today. Thanks a ton!

**Comment by bevanjkay** [comment]
> Do you think I should've created a new cask? As the older version hasn't gotten (and won't) get any updates in the last 5 years.
> I can also probably ask her to move it to the organization of @therehq if it's a must.

We do need some kind of official information to suggest that the original cask should be replaced with this.
I can't see a recommendation on the old GitHub repository, that this is to become a successor, and the homepage url from the repository goes to a different website.

See our docs on forks here, which would essentially apply in this instance also;
https://docs.brew.sh/Acceptable-Casks#forks-and-apps-with-conflicting-names

File: Casks/t/there.rb [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
+      verified: "github.com/dena-sohrabi/There/"
   name "There"
-  desc "Tool to display the local times of people in a team"
+  desc "Tool to display the local times of friends, teammates, cities or any time zone"
   homepage "https://there.pm/"
 
   livecheck do
-    url :url
-    strategy :github_latest
+    strategy :github_releases
```
Comment (samford) [inline Casks/t/there.rb:12]:
```suggestion
    url :url
    strategy :github_latest
```

This should be restored to `GithubLatest`, as the `GithubReleases` strategy isn't necessary here. `GithubLatest` checks the "latest" release on GitHub, whereas `GithubReleases` checks multiple recent releases (so it has to transfer more data, adding to overhead). We only use `GithubReleases` when checking the "latest" release doesn't work as expected (and we don't have other options).

Besides that, all the existing `livecheck` strategies other than `ExtractPlist` require a `url` value, so it shouldn't be omitted here.

**Comment by morajabi** [comment]
@bevanjkay I've added the correct notes and website on the original repo: https://github.com/therehq/there-desktop/tree/master

File: Casks/t/there.rb (hunk 2 above)
Comment (morajabi) [reply Casks/t/there.rb]: Thanks for the review, I applied your suggestion.

**Comment by morajabi** [comment]
I really appreciate your work here, I have deep respects for open source maintainers who give their time away for free ❤️ I hate to be that guy that pushes people on a PR tbh, but since the new app is getting unexpected attention [on X](https://x.com/morajabi/status/1835983451779953006) and [Product Hunt](https://www.producthunt.com/posts/there-2) people are [installing the previous version](https://x.com/morajabi/status/1836325548827508845) and [asking](https://x.com/JulienBlanchon/status/1836149757661598022) for the [new one on brew](https://x.com/__2024__2025/status/1836128550195499426). It'd be amazing if we can push this update out as soon as you think it's possible. Thank you

**Comment by julien-blanchon** [comment]
What's blocking this PR ?

**Review by bevanjkay — APPROVED** [review]

**Comment by morajabi** [comment]
Thanks @bevanjkay!

**Comment by bevanjkay** [comment]
Thank you for your patience. Our team of volunteer maintainers are distributed throughout many timezones, but there isn't always someone available for a review.

**Pull request merged**